src/utils/cdn-utils.ts

```diff
@@ -20,47 +20,47 @@ export const packDownloadRequest = (request: CDNDownloadDataRequest): Buffer =>
 export const unpackDownloadRequest = (buf: Buffer): CDNDownloadDataRequest => {
```

Comment:
We need to keep the order of the parameters, this file needs a revert.

Reply:
u mean the file `cdn-utils.spec.ts`?